src/main/java/ch/hearc/cafheg/infrastructure/persistance/AllocataireMapper.java

```diff
@@ -13,65 +15,68 @@
 
 public class AllocataireMapper extends Mapper {
 
+    private static final Logger log = LoggerFactory.getLogger(AllocataireMapper.class);
```

Comment:
You named the variable `logger` before, and now `log`, not mandatory but I would suggest to make the variable name consistent across files.